### PR TITLE
Exposing job's  scheduled unit and interval

### PR DESCRIPTION
**Pull request by chicknsoup**
### What does this do?
Exposes job's scheduled unit and interval
Added String() function to schedulingUnit 

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No need

### Did you document any new/modified functionality?
Yes

### CODE REVIEW

File: job.go [hunk 1]
```diff
@@ -446,6 +446,16 @@ func (j *Job) ScheduledTime() time.Time {
 	return j.nextRun
 }
 
+// ScheduledUnit returns the scheduled unit of the Job.
+func (j *Job) ScheduledUnit() schedulingUnit {
```
Comment (JohnRoesler) [inline job.go]: there should be a public version of schedulingUnit - or perhaps just make `schedulingUnit` and all the const values public

File: job.go [hunk 2]
```diff
@@ -446,6 +446,16 @@ func (j *Job) ScheduledTime() time.Time {
...
+        return j.unit
```
Comment (JohnRoesler) [inline job.go]: please run `gofumpt -w .`

Comment (chicknsoup) [reply job.go]: Done, thank you

File: job.go (hunk 1 above)
Comment (chicknsoup) [reply job.go]: Changed the return type to string. I think that should be enough.

**Review by JohnRoesler — APPROVED** [review]

**Pull request merged**